apps/flame-defi/app/components/WithdrawCard/WithdrawCard.tsx

```diff
@@ -472,40 +479,43 @@ export default function WithdrawCard(): React.ReactElement {
         </div>
       </div>
 
-      <div className="is-flex is-flex-direction-row is-align-items-center">
-        <div className="card-spacer" />
+      <div className="flex items-center">
+        <div className="w-full border-t border-grey-dark my-4" />
       </div>
 
-      <div className="field">
-        <div className="is-flex is-flex-direction-row is-align-items-center">
-          <div className="label-left">Amount</div>
-          <div className="control mt-1 is-flex-grow-1">
-            <input
-              className="input is-medium"
-              type="text"
-              placeholder="0.00"
-              onChange={updateAmount}
-              value={amount}
-            />
+      <div className="mb-4">
+        <div className="flex flex-col">
+          <div className="mb-2 sm:hidden">Amount</div>
+          <div className="flex flex-col sm:flex-row sm:items-center">
+            <div className="hidden sm:block sm:mr-4 sm:min-w-[60px]">
+              Amount
+            </div>
+            <div className="flex-grow">
+              <input
+                className="w-full p-3 bg-transparent border border-grey-dark focus:border-white focus:outline-none rounded-xl text-white text-[20px]"
+                type="text"
+                placeholder="0.00"
+                onChange={updateAmount}
+                value={amount}
+              />
+            </div>
           </div>
+          {!isAmountValid && hasTouchedForm && (
+            <div className="mt-2 text-red-500 text-sm">
+              Amount must be a number greater than 0
+            </div>
+          )}
         </div>
-        {!isAmountValid && hasTouchedForm && (
-          <div className="help is-danger mt-2">
-            Amount must be a number greater than 0
-          </div>
-        )}
       </div>
 
-      <div className="card-footer mt-4">
-        <button
-          type="button"
-          className="button is-tall is-wide has-gradient-to-right-orange has-text-weight-bold has-text-white"
+      <div className="mt-4">
+        <ActionButton
           onClick={handleWithdraw}
           disabled={isWithdrawDisabled}
-        >
-          {isLoading ? "Processing..." : "Withdraw"}
-        </button>
+          isLoading={isLoading}
+          buttonText={"Withdraw"}
+        />
       </div>
     </div>
   );
-}
+}
```

Comment:
Can you run the `format` command? I'll make a task for adding a Github action that will fail when formatting isn't done.